components/layouts/DocsLayout.js

```diff
@@ -14,7 +14,7 @@ export default function DocsLayout({ children, title }) {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <Page>
-        <h1 className="text-4xl mb-4 font-bold">Documentation</h1>
+        <h1 className="text-4xl mb-4 font-bold max-[293px]:text-xl">Documentation</h1>
```

Comment:
Can we use Tailwind sizing, not a new size

Reply:
As you know that Responsive Breakpoint "sm"  works only when you have min-width:640px but here you can see when I shrink the screen is near 274px "Document" styling is bigger so it takes more space and above the navigation bar breaks 
So for that reason, I have used tailwinds  [Arbitrary values](https://tailwindcss.com/docs/responsive-design#arbitrary-values) ie only bringing down the font size of the "Document" when width<293px. 
screen width above 293px text-4xl does a good job but fails at less than 293px.

![image](https://user-images.githubusercontent.com/52135949/215961746-1b6ae5ec-30d6-4773-b330-9af4be31b012.png)